.github/workflows/test.yml

```diff
@@ -1,8 +1,6 @@
 name: Test
 
 on:
-  push:
```

Comment:
As you change this section, should we run the test for all `push` instead of `pull_request`? Then every time we push any code, it will be run in advance. What do you think?